recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
```

Comment:
I don't think that would work for a generic git repo, including those hosted locally.  Might be worth having a `github: org/repo` key though...